### DISCUSSION TITLE
Support emoji shortcode

### DISCUSSION THREAD

**Discussion by azzamsa** [usememos/memos#3566] [Ideas]
Hi, 👋🏽 

It would be cool if we have emoji completion using `:` colon.
- https://emojipedia.org/shortcodes

Similar request, to see it in action: https://github.com/microsoft/vscode-pull-request-github/issues/4994

Github, WhatsApp, Discord, etc had this feature implemented already.

Thanks 🍜

**Comment by adoyle-h** [comment]
+1
